gulpfile.ts

```diff
@@ -42,10 +39,28 @@ gulp.task('lint', () => {
 });
 
 gulp.task('lint:fix', () => {
-  gulp.src([
-    './server/**/*.ts',
-    './client/**/*.ts'
-  ])
+  gulp.src('./server/**/*.ts')
+  .pipe(tslint({
+    fix: true,
+    formatter: 'codeFrame'
+  }))
+  .pipe(tslint.report({
+    summarizeFailureOutput: true
+  }));
+});
+
+gulp.task('lint_client', () => {
```

Comment:
Let's leave a FIXME comment to remember to remove this command later.
(If possible, please file a issue as well.)